src/Compilers/CSharp/Portable/Syntax/Syntax.xml

```diff
@@ -1121,7 +1121,7 @@
   </Node>
   <Node Name="DeclarationExpressionSyntax" Base="ExpressionSyntax">
     <Kind Name="DeclarationExpression"/>
-    <Field Name="Declaration" Type="VariableDeclarationSyntax">
+    <Field Name="VariableComponent" Type="VariableComponentSyntax">
```

Comment:
If I understand correctly, this will currently only ever be a `TypedVariableComponentSyntax`, but we're leaving it purposefully open. Is that right?